### PR TITLE
Added How to Create a List Challenge Solutions

### LINKED ISSUE #6: Add Challenge Link for "How to Create a List"
Here's the source article: https://therenegadecoder.com/code/how-to-create-a-list-in-python/

Here's a quick summary of the challenge: 

> create a list which contains the first 100 terms of the fibonacci sequence

For this, I'd be content with the solution being written and added to the challenges folder as Python source code. If you want to go the extra mile, you can use [Carbon](https://carbon.now.sh/) to create an image of the code snippet to post on Twitter with the hashtag [#RenegadePython[(https://twitter.com/hashtag/RenegadePython). If you do that, you can link the tweet in the README, and I'll share your tweet in the article.

### PR DESCRIPTION
**Pull request by Muhimen123**
In this script, three types of solutions were introduced.
- using for loop
- using recursion
- using list comprehensions

### CODE REVIEW

File: challenges/how-to-create-a-list.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+def using_normal_loops():
+    fibonacci = [1, 1]
+    for i in range(2, 100):
+        previous_number_1 = fibonacci[i - 1]
+        previous_number_2 = fibonacci[i - 2]
+        new_number = previous_number_1 + previous_number_2
+        fibonacci.append(new_number)
+
+    for idx, val in enumerate(fibonacci):
```
Comment (jrg94) [inline challenges/how-to-create-a-list.py]: Very cool! I think this function should probably just return the list (if we're following the challenge strictly). That said, this looks great.

File: challenges/how-to-create-a-list.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# The challenge is to write a script that can generate all the fibonacci numbers form 1 to 100
```
Comment (jrg94) [inline challenges/how-to-create-a-list.py]: Can we separate these three solutions into three files, and place each of them in a folder titled "how-to-create-a-list"? I think that'll make this more consistent with the other challenge.

File: challenges/how-to-create-a-list.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+using_normal_loops()
+for i in range(101):
+    print(f"{i+1}: {using_recursion(i)}")
+
+# Using list comprehension
+    # This is nothing different than the first one. But in a shorter manner
+
+fibonacci = [1, 1]
+[fibonacci.append(fibonacci[i - 1] + fibonacci[i - 2]) for i in range(2, 100)]
```
Comment (jrg94) [inline challenges/how-to-create-a-list.py]: Interesting solution! I generally don't love list comprehensions being used this way. After all, it's being used to update an external list, but a list is still generated (filled with None). That said, it gets the job done! 👍

**Review by jrg94 — CHANGES_REQUESTED** [review]
Overall, I'm really thankful for this addition. I just have a couple requests. Let me know what you think.

File: challenges/how-to-create-a-list.py (hunk 1 above)
Comment (Muhimen123) [reply challenges/how-to-create-a-list.py]: Sure, why not?

File: challenges/how-to-create-a-list.py (hunk 2 above)
Comment (Muhimen123) [reply challenges/how-to-create-a-list.py]: So, you want the file structure to look like this?
```
./how-to-create-a-list
    /loop.py
    /recursion.py
    /list comprehensio.py
```

No big deal.

File: challenges/how-to-create-a-list.py (hunk 3 above)
Comment (Muhimen123) [reply challenges/how-to-create-a-list.py]: I also don't like list comprehensions like this. But just to prove, "If you try, you can solve any problem."
😉

File: challenges/how-to-create-a-list.py (hunk 2 above)
Comment (jrg94) [reply challenges/how-to-create-a-list.py]: Yes! That would be great. Thanks.

File: challenges/how-to-create-a-list.py (hunk 3 above)
Comment (jrg94) [reply challenges/how-to-create-a-list.py]: Hahah yeah, fair enough! I'm happy with it if you are.

**Comment by Muhimen123** [comment]
Changes are made. You can take a look now. 😊

**Review by jrg94 — APPROVED** [review]
Looks good to me! I'm going to post one of these and credit you on Twitter (pending your approval). Do you have a profile you'd like me to link?

**Comment by Muhimen123** [comment]
Unfortunately, I don't have a Twitter account. 😕

**Comment by jrg94** [comment]
That's no problem! I meant like I'll post it, and I'll link to one your profiles (DEV, GitHub, etc.).

**Comment by Muhimen123** [comment]
Ow... Yeah, you can link my DEV or Github. I have no problem on that.

**Pull request merged**